ar-001/auth.json

```diff
@@ -0,0 +1,283 @@
+        },
+        "authenticator_app_manual_page": {
+            "page_title": "تطبيق المصادقة |  ${business_name}",
+            "heading": "تطبيق المصادقة",
+            "description": "أدخل الرمز التالي في تطبيق المصادقة المفضل لديك.  ثم أدخل رمز الاستخدام لمرة واحدة الذي ستتلقاه أدناه.",
+            "code_input_label": "رمز الإستخدام لمرة واحدة ",
+            "code_input_error_required": "الرجاء إدخال رمز التحقق",
+            "code_input_error_incorrect_code_length": "يجب أن يتكون رمز التأكيد من 6 أرقام",
+            "code_input_error_incorrect_code": "الرجاء إدخال رمز تأكيد صحيح",
+            "code_input_error_expired_code": "لقد انتهت صلاحية رمز التأكيد الخاص بك، يرجى طلب رمز جديد",
+            "continue_button": "المتابعة",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "mfa_sms_page": {
+            "page_title": "أدخل رقم الهاتف |  ${business_name}",
+            "heading": "المصادقة عبر الرسائل القصيرة",
+            "description": "أضف رقم هاتف لتلقي الرموز عبر رسالة نصية",
+            "country_input_label": "الدولة او المنطقة",
+            "country_input_error_invalid": "مطلوب دولة صحيحة",
+            "phone_input_label": "رقم الهاتف",
+            "phone_input_error_required": "أدخل رقم الهاتف",
+            "phone_input_error_invalid_format": "أدخل الأرقام فقط بدون مسافات، باستثناء الصفر البادئ",
+            "phone_input_error_invalid_region": "الرقم غير صالح لهذه المنطقة",
+            "phone_input_help_text": "اترك الصفر البادئ",
+            "continue_button": "المتابعة",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "sms_verify_code_page": {
+            "page_title": "مصادقة متعددة العوامل |  ${business_name}",
+            "heading": "تحقق من رقم الهاتف",
+            "description": "أدخل الرمز الذي أرسلناه للتو إلى ${phone_number]",
+            "code_input_label": "رمز",
+            "code_input_error_expired_code": "لقد انتهت صلاحية رمز التأكيد الخاص بك، يرجى طلب رمز جديد",
+            "code_input_error_incorrect_code": "الرجاء إدخال رمز تأكيد صحيح",
+            "code_input_error_incorrect_code_length": "يجب أن يتكون رمز التأكيد من 6 أرقام",
+            "code_input_error_resend_requested_too_soon": "لا يمكن إرسال رمز آخر بعد.  الرجاء إعادة المحاولة في وقت لاحق",
+            "confirmation_code_input_error_unknown_error": "خطأ في المصادقة",
+            "continue_button": "تحقق",
+            "didnt_receive_a_code_description": "لم تستلم رمز ؟",
+            "didnt_receive_a_code_link": "إعادة إرسال الرمز ",
+            "choose_another_method_link": "إختر طريقة أخرى"
+        },
+        "verification_code_sms": {
+            "description": "\"${generated code} هو الرمز الذي تستخدمه لمرة واحدة لتسجيل الدخول إلى ${business_name} @ ${receiver_domain} # ${generated code]\""
+        },
+        "choose_organization_page": {
+            "page_title": "اختر مؤسسة |  ${business_name}",
```

Comment:
@onderay there looks like additional whitespace here on the right hand side of ` | `

Reply:
@DaveOrDead updated